docs/fastdds/xml_configuration/common.rst

```diff
@@ -68,11 +68,11 @@ The table presented below outlines each possible Locator's field.
   * - ``<port>``
     - RTPS port number of the locator. |br|
       *Physical port* in UDP, *logical port* in TCP.
-    - ``uint32_t``
+    - ``uint16_t``
```

Comment:
Do we also need to change it [here](https://fast-dds.docs.eprosima.com/en/latest/fastdds/transport/transport_api.html#id2)?